src/controllers/loader/index.ts

```diff
@@ -0,0 +1,23 @@
+import { ResourceStoreContext } from '../../common/types';
+import { combine } from '../../common/utils/combine';
+import { entryPointsLoader } from '../../entry-points/loader';
+import { resourcesLoader } from '../../resources/loader';
+
+type Props = {
+  context: ResourceStoreContext | undefined;
+  resourceData: any;
+  timeout?: number;
+  isStatic?: boolean;
+};
+
+export const createCombinedLoader = (props: Props) => {
```

Comment:
Was this intended to be part of the public API? Because imo entrypoints shouldn't be in here. I was thinking that the `Router` can take in a `loader` or `loaders` (if you follow Alberto's suggestion) that defaults to the resource loader, making it a non-breaking change. Then consumers that want to combine them just opt into it through the new API. That way you separate the `Props` from `createCombinedLoader` into a `createResourceLoader`